deployment/ansible/roles/model-my-watershed.rwd/tasks/configuration.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Configure application
+  copy: content="{{ item.value }}"
+        dest="{{ envdir_home }}/{{ item.key }}"
+        owner=root
+        group=mmw
+        mode=0750
+  with_dict: "{{ app_config }}"
```

Comment:
I considered adding a "Restart celery" notify handler here, but wasn't sure if raising a handler from another role is the correct approach.